packages/create-universal-package/lib/tasks/clean.js

```diff
@@ -0,0 +1,11 @@
+const cp = require('child_process');
+const {promisify} = require('util');
+const exec = promisify(cp.exec);
+const path = require('path');
+
+module.exports = function clean(opts) {
+  return Promise.all([
+    exec(`rm -rf ${path.join(opts.dir, 'dist-tests/')}`),
```

Comment:
NOTE: If we really care about good cross OS support this should probably use something like rimraf

Reply:
rimraf is already a dependency, something went weird in https://github.com/rtsao/create-universal-package/commit/110e3bf0df943dc12b189fc053d90b920561d780 and it got lost. I've updated your fork to use rimraf again